build.gradle

```diff
@@ -41,10 +41,10 @@ project.version = "$gradle.ext.version_number-$build_number"
 def dependencyVersions = [
         opensaml:"$opensaml",
         dropwizard: '1.3.5',
-        ida_utils: '370',
+        ida_utils: '386',
         ida_test_utils: '44',
         dev_pki: '1.1.0-34',
-        saml_libs_version: "$opensaml-224"
+        saml_libs_version: "$opensaml-SNAPSHOT"
```

Comment:
I'm sure this is just a miss, we shouldn't use SNAPSHOT

Reply:
Yeah, this is just hear so it works locally with my version of saml-lib. Once I've got the PR there merged I'll be updating this with the new build number